src/configuration/app-containers.md

```diff
@@ -467,17 +468,17 @@ hooks:
 
 ### Crons / Cron jobs
 
-Cron jobs allow you to run scheduled tasks at specified times or intervals. The `crons` object describes these tasks and the schedule that they are triggered on.  Each item in the list is a unique name identifying a separate cron job.
+Cron jobs allow you to run scheduled tasks at specified times or intervals. The `crons` object describes these tasks and the schedule when they are triggered.  Each item in the list is a unique name identifying a separate cron job.
 
-It has a few subkeys which are:
+It has a few subkeys listed below:
 
 -   **spec**: The [cron specification](https://en.wikipedia.org/wiki/Cron#CRON_expression). For example: `*/20 * * * *` to run every 20 minutes.
 -   **cmd**: The command that is executed, for example
     `cd public ; drush core-cron`
 
-The minimum interval between cron runs is 5 minutes, even if specified as less.
+The minimum interval between cron runs is 5 minutes; even if specified as less.
 
-Note that cron runs are executed using the dash shell, not the bash shell used by normal SSH logins. In most cases that makes no difference but may impact some more involved cron scripts.
+Note that cron runs are executed using the dash shell; not the bash shell used by normal SSH logins. In most cases, that makes no difference but may impact some more involved cron scripts.
```

Comment:
Same.  The comma is better in the first case as it's a sentence fragment otherwise, and unnecessary in the latter case.  This line can be stet.